users/autostart.rst

```diff
@@ -17,6 +17,17 @@ Windows
 There is currently no official installer available for Windows. However,
 there are a number of easy solutions.
 
+Built-in solutions:
+
+- `Run at user log on or at system startup using Task Scheduler
+  <#task-scheduler>`__
+- `Run at user log on using Startup folder <#start-on-login>`__
```

Comment:
I think these should be rst-style references, not based on the HTML anchors. As is, this will break in pdf and man page exports etc. Otherwise LGTM

Reply:
Let me fix them. I am not very accustomed to the whole RST formatting yet. I would like to finish https://github.com/syncthing/docs/pull/515 first though, as this one is somewhat co-dependent. I will try to do both this week.

Reply:
I think that I have fixed the URLs. Please check and feel free to merge the commit if everything is OK :slightly_smiling_face:.